src/main/kotlin/app/revanced/patches/layout/ForceSettingsPatch.kt

```diff
@@ -0,0 +1,68 @@
+package app.revanced.patches.layout
+
+import app.revanced.patcher.PatcherData
+import app.revanced.patcher.extensions.addInstructions
+import app.revanced.patcher.extensions.or
+import app.revanced.patcher.patch.*
+import app.revanced.patcher.signature.MethodMetadata
+import app.revanced.patcher.signature.MethodSignature
+import app.revanced.patcher.signature.MethodSignatureMetadata
+import app.revanced.patcher.signature.PatternScanMethod
+import app.revanced.patcher.smali.toInstructions
+import org.jf.dexlib2.AccessFlags
+import org.jf.dexlib2.Opcode
+
+private val compatiblePackages = listOf(
+    PackageMetadata(
+        "com.google.android.youtube",
+        listOf("17.14.35")
+    )
+)
+
+class ForceSettingsPatch : Patch(
```

Comment:
This patch should have a different name. Something like "DownloadPatch" or "DownloadAndBackgroundPlaybackPatch".